spec/internal/module.lua

```diff
@@ -6,6 +6,34 @@ local function reload(name)
 end
 
 
+local reload_helpers
+do
+  local sys = require("spec.internal.sys")
+
+  -- flavor could be "traditional","traditional_compatible" or "expressions"
+  -- changing flavor will change db's schema
+  reload_helpers = function(flavor)
+    _G.kong = {
+      configuration = {
+        router_flavor = flavor,
+      },
+    }
+
+    sys.setenv("KONG_ROUTER_FLAVOR", flavor)
+
```

Comment:
I see that the new implementation of `reload_helpers` is missing the following lines compared to the previous one, and I'm not sure what the potential impact would be.

```lua
    package.loaded["spec.helpers"] = nil
    package.loaded["kong.global"] = nil
    package.loaded["kong.cache"] = nil
    package.loaded["kong.db"] = nil
```

Reply:
```lua
package.loaded["kong.global"] = nil
package.loaded["kong.cache"] = nil
```
It seems that these two don't have an impact, but it's better to keep them.

Reply:
done.